x/relationships/module.go

```diff
@@ -146,29 +147,43 @@ func (am AppModule) EndBlock(ctx sdk.Context, _ abci.RequestEndBlock) []abci.Val
 //____________________________________________________________________________
 
 // AppModuleSimulation defines the module simulation functions used by the profile module.
-type AppModuleSimulation struct{}
+type AppModuleSimulation struct {
```

Comment:
All the changes here should be reverted. No need to declare a new `AppModuleSimulation` type when the `AppModule` already implements this

Reply:
I used this `AppModuleSimulartion` to be able to pass the `fees` keeper to the sims tests without touching the `AppModuleBasic` type.